lib/cisco_node_utils/interface_ospf.rb

```diff
@@ -23,53 +23,62 @@
 module Cisco
   # InterfaceOspf - node utility class for per-interface OSPF config management
   class InterfaceOspf < NodeUtil
-    attr_reader :interface, :ospf_name
+    attr_reader :intf_name, :ospf_name, :area
+    attr_accessor :get_args
 
-    def initialize(int_name, ospf_name, area, create=true)
-      fail TypeError unless int_name.is_a? String
+    def initialize(intf_name, ospf_name, area, create=true)
+      fail TypeError unless intf_name.is_a? String
       fail TypeError unless ospf_name.is_a? String
       fail TypeError unless area.is_a? String
-      fail ArgumentError unless int_name.length > 0
+      fail ArgumentError unless intf_name.length > 0
       fail ArgumentError unless ospf_name.length > 0
       fail ArgumentError unless area.length > 0
 
       # normalize
-      int_name = int_name.downcase
-      @interface = Interface.interfaces[int_name]
-      fail "interface #{int_name} does not exist" if @interface.nil?
+      @intf_name = intf_name.downcase
+      fail "interface #{@intf_name} does not exist" if
+        Interface.interfaces[@intf_name].nil?
```

Comment:
This is probably a problem - because it's going to get all interfaces by default. Let me see if there's a solution here.

Reply:
Latest commit addresses this issue. You can continue the review now.